stock_exchange/apps/stock/models.py

```diff
@@ -1,4 +1,77 @@
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from datetime import date
 
+from stock_exchange.apps.users.models import User
 
-# Add your models here
\ No newline at end of file
+#This tuple is used for choice option for
+#inform users about market operation status.
+MARKET_OPTIONS = {
+    ('B', 'buy'),
```

Comment:
I would rather make another file like `choices.py` for those even to it's only one tuple now.

Reply:
Maybe add translation here also, if you will use it in templates. If not you can omit this